### ISSUE TITLE
`indentLogicalExpressions` is not being accepted by eslint

### ISSUE THREAD

**Issue by swernerx** [yannickcr/eslint-plugin-react#1083]
When adding the third parameter to jsx-indent like this, it seems like Eslint is not happy about it:

```
Error: /Users/sebastian/Workspace/sebastian-software/readable-code/eslint/react.yml:
        Configuration for rule "react/jsx-indent" is invalid:
        Value "2,[object Object]" has more items than allowed.
```

Config:

```yaml
rules:
  react/jsx-indent:
    - error
    - 2
    - indentLogicalExpressions: true
```

**Comment by jomasti** [comment]
It looks like the changes that introduced for that rule were merged in just after the latest release, so that option won't work just yet.

**Comment by fmoliveira** [comment]
Hey @jomasti I'm updating from version 6.10.2 to 7.8.2 and I'm wondering what's happened to the parameter `indentLogicalExpressions`, which isn't accepted in the `react/jsx-indent` rule anymore. I haven't found any sign of neither in the docs or in the source code.

*Question*: Was this parameter discontinued, or replaced for another rule?

*My issue:* We had it set to `false` in version 6.10.2 because of #1117 and we're afraid the problem happens again if this parameter had been replaced for another rule.

**Comment by ljharb** [comment]
It was reverted in https://github.com/yannickcr/eslint-plugin-react/commit/e9fb52d332e15686865952371c333b19af13eede#diff-aca6d5d9aa8669fa64844849538bc0a0 And nobody’s tried to add it back yet.

Since it was a bug - a temporary one - it seems a silly thing to be afraid of. You can upgrade and remove the parameter and you’ll be fine.

**Comment by fmoliveira** [comment]
Thanks a million, @ljharb ! :)

I've seen today that it was fixed in 6.10.3 but we never updated it because we were caught in a long project rush so it's very relieving to hear that from you. Super thanks again! 😃
